.storybook-s2/docs/Migrating.jsx

```diff
@@ -146,7 +149,9 @@ export function Migrating() {
         </ul>
 
         <H3>IllustratedMessage</H3>
-        <P>No updates needed.</P>
+        <ul className="sb-unstyled">
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Update illustrations to be from <Code>@react-spectrum/s2/illustrations</Code>. See <Link href="?path=/docs/illustrations--docs">Illustrations</Link></li>
```

Comment:
Looks like this link doesn't quite work because it changes the path to contain the iframe